addons/campsite/constants.hpp

```diff
@@ -0,0 +1 @@
+#define CAMPSITE_BASIC_ARRAY ["coopr_item_camping"]
```

Comment:
Start CoopR Vars alwaywas with `COOPR_`. It's name "campsite basic array" is a bit too common. You mean like "campsite camp items" ?